st2client/st2client/commands/resource.py

```diff
@@ -198,10 +197,12 @@ def get_resource_by_pk(self, pk, **kwargs):
         try:
             instance = self.manager.get_by_id(pk, **kwargs)
         except Exception as e:
-            traceback.print_exc()
+            # traceback.print_exc()
```

Comment:
@ashwini-orchestral Please remove this line instead of commenting it out. It's a bad practice and no one will come back and clean this up.